cli/enable/enable_test.go

```diff
@@ -0,0 +1,67 @@
+package enable
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/otiai10/copy"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+)
+
+func TestEnableBase(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "enable_test")
+	defer os.RemoveAll(tempDir)
+	assert.Nil(t, err)
+	err = copy.Copy("./test_data", tempDir)
```

Comment:
This is a very small env. It would be nice, if we will generate it in-place during the test. Not required.
Also, the directory name should be `testdata`.

Reply:
I don't think that we need a bigger env, it won't change anything.

Reply:
"small env" is good in this case. I'm talking about possibility to generate such small envs on the fly. Not about making them bigger.